coderedcms/static/js/codered-front.js

```diff
@@ -216,4 +216,24 @@ $(document).ready(function()
             $(this).modal('show');
         }
     });
-});
\ No newline at end of file
+
+
+    /*** Tracking ***/
+    $('a').on('click', function(){
+        if($("meta[name='tracking:click']").length){
```

Comment:
I would reverse the `if` and the `a.onclick` nesting so that the event is only wired up if we are going to track in the first place, otherwise every `click` would trigger an event that does nothing.